.github/actions/update-config/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Config
+description: 'Updates the configuration'
+
+inputs:
+  host-name:
+    description: 'The host name to update'
+    required: true
+  android-facet-id:
+    description: 'The android facet id to update'
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Update Configuration (loginRequestURL)
+      uses: jossef/action-set-json-field@v2.1
+      with:
+        file: assets/config_authentication_cloud.json
+        field: login.loginRequestURL
+        value: "https://${{ inputs.host-name }}.mauth.nevis.cloud/_app/auth/pwd"
+
+    - name: Update Configuration (hostname)
+      uses: jossef/action-set-json-field@v2.1
+      with:
+        file: assets/config_authentication_cloud.json
+        field: sdk.hostname
+        value: "${{ inputs.host-name }}.mauth.nevis.cloud"
+
+    - name: Update Configuration (facetId)
+      if: ${{ inputs.android-facet-id }}
+      uses: jossef/action-set-json-field@v2.1
+      with:
+        file: assets/config_authentication_cloud.json
```

Comment:
Isn't it an iOS configuration file? Why do we update it with Android Facet ID?

Reply:
Nevermind, I realized it is the Flutter app, so the config file contains android part as well.